### ISSUE TITLE
Clarification on what python files to run and with what arguments.

### ISSUE THREAD

**Issue by zenonasz** [kessoning/Audio-Offline-Analysis#1]
Hi, 

Overall you have a very impressive portfolio there, great work. 

The documentation needs some clarification. There are two Python files the beat_detection.py and the audio_to_params.py. 
beat_detection.py has no --formula argument but has the --mul argument is it the very same parameter? can you please give some details shall we run both the beat_detection.py and the audio_to_params.py and use both output.txt?

Can you please also give an example of how to use it in processing the example you provided is not clear enough.

Regards and keep up the good work, again impressive porfolio!

**Comment by kessoning** [comment]
Hi @zenonasz ,

Thanks for the feedback. There were some things I didn't see in the repository, which I just fixed.

I made a friendly Processing example (including a class that can be easily copy-pasted and implemented in other projects).

In the description, I explain the usage of the formula. I also removed completely it from the beat_detection (the --mul was the multiplier of the value) as it was completely useless (making it easier to use).
